src/views/Game/AIReviewChart.tsx

```diff
@@ -284,11 +251,23 @@ export class AIReviewChart extends React.Component<AIReviewChartProperties, any>
     plot() {
         let entries:Array<AIReviewEntry>;
 
+        let use_score_safe = this.props.use_score
+          && this.props.ai_review.scores != null
+          && !this.props.ai_review.scores.includes(0);
+
         if (this.props.entries.length > 0) {
-            entries = this.props.entries.map(x => x);
-            entries.unshift({win_rate: 0.5, move_number: 0, num_variations: 0});
+            entries = this.props.entries.map((x, i) => {
+                return {
+                    win_rate: x.win_rate,
+                    score: x.score === 0 && use_score_safe ? this.props.ai_review.scores[i] : x.score,
```

Comment:
The fact that `entries*.score` and `ai_review.scores` could be different, was honestly really confusing.